internal/podman/client.go

```diff
@@ -316,11 +325,14 @@ func (o *client) runPodman(ctx context.Context, arg ...string) ([]byte, error) {
 
 	if o.debug {
 		_, _ = o.outWriter.Write(output)
+	}
 
-		var exitErr *exec.ExitError
-		if errors.As(err, &exitErr) {
+	var exitErr *exec.ExitError
+	if errors.As(err, &exitErr) {
+		if o.debug {
 			_, _ = o.outWriter.Write(exitErr.Stderr)
 		}
+		err = extractErrorMessage(exitErr)
```

Comment:
was this line lost after merging master? `was this line lost after merging master?
`err = fmt.Errorf("%w: you may rerun with this command with the flag '--debug' to gather more information", err)`

should this be?
```suggestion
		err = was this line lost after merging master?
`err = fmt.Errorf("%w: you may rerun with this command with the flag '--debug' to gather more information", extractErrorMessage(exitErr))`
```